install.ps1

```diff
@@ -0,0 +1,107 @@
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+
+$ProgressPreference = 'SilentlyContinue'
+$release = Invoke-RestMethod -Method Get -Uri "https://api.github.com/repos/ducaale/xh/releases/latest"
+$asset = $release.assets | Where-Object name -Like *.zip
+$destDir = "$home\bin\"
+$zipFile = "$env:TEMP\$($asset.name)"
+
+Write-Output "Downloading: $($asset.name)"
+Invoke-RestMethod -Method Get -Uri $asset.browser_download_url -OutFile $zipFile
+
+# checks if an older version of xh.exe exists in '$destDir', if yes, then delete it, if not, then download latest zip to extract from
+$xhPath = "${destDir}xh.exe"
+if (Test-Path $xhPath -PathType Leaf) {
+	"`n xh.exe exists in $destDir, deleting..."
+	rm -r -fo $xhPath
+} else {
+	"`n ${destDir}xh.exe does not exist"
+}
+
+#xh.exe extraction start
+Add-Type -Assembly System.IO.Compression.FileSystem
+
+$zip = [IO.Compression.ZipFile]::OpenRead($zipFile)
+$entries=$zip.Entries | Where {$_.FullName -Like '*.exe'}
+
+#create dir for result of extraction
+New-Item -ItemType Directory -Path $destDir -Force
+
+#extraction
+$entries | foreach {[IO.Compression.ZipFileExtensions]::ExtractToFile( $_, $destDir + $_.Name) }
+
+#free zipfile
+$zip.Dispose()
+
+Remove-Item -Path $zipFile
+
+Write-Host "`n Extracted 'xh.exe' file to $destDir"
```

Comment:
xh supports [xhs symlink](https://github.com/ducaale/xh#xh-and-xhs), which will default to HTTPS requests. Can you create that here?

Reply:
Sorry about the late reply to all comments. This one is a tricky one. 
1. Powershell needs to be run as an administrator for symbolic link creation. 
2. Not gonna lie, I'm not sure if its working. I added an image to this comment to get confirmation

![mstsc_4RvYD2bUhn](https://user-images.githubusercontent.com/73487474/149366275-cc883289-6589-4d36-9a68-2037fb0d2125.png)
.

Reply:
Maybe it would be better to just copy the executable?

A shortcut [could also work](https://stackoverflow.com/a/30921867), but the user would have to type out `xhs.lnk` (unless we modify `PATHEXT` but that seems too intrusive).

What do the package managers do?

Reply:
Scoop copies the binary (I think). See https://github.com/ScoopInstaller/Main/blob/5180fcd99cc1c1453302da1d0402e1bd7f3362bf/bucket/xh.json#L17.

>Not gonna lie, I'm not sure if its working. I added an image to this comment to get confirmation

![Inked149366275-cc883289-6589-4d36-9a68-2037fb0d2125](https://user-images.githubusercontent.com/16341131/149384602-b016995d-c070-47fd-9dbc-ee5fdc8cf393.jpg)

Reply:
I used `cmd /c mklink /d xhs $xhPath` in install.ps1. It launches cmd and uses cmd's mklink command, because powershell's junction did not work for me. Sending a new pull request with the updated powershell file, if I succeed. If not il try to check back tomorrow (16 jan).